database/sql/headers.go

```diff
@@ -187,11 +193,27 @@ func (h *HeadersDb) Create(ctx context.Context, req dto.DbBlockHeader) error {
 		_ = tx.Rollback()
 	}()
 	if _, err := tx.NamedExecContext(ctx, sqlInsertHeader, req); err != nil {
+		if isUniqueViolation(err) {
```

Comment:
Are you sure we will get this error here, not on `tx.Commit()` ?

Reply:
Yes, 100% sure